services/IBF-pipeline/pipeline/runSetup.py

```diff
@@ -2,6 +2,7 @@
 from lib.setup.createSubFolders import createSubFolders
 from lib.setup.staticDataDb import uploadStaticToDb, processStaticDataDb
 from settings import *
+from secrets import *
```

Comment:
@jannisvisser is the `runSetup.py` file still needed? If not, can we delete the file?